addons/interaction/functions/fnc_renameGroup.sqf

```diff
@@ -0,0 +1,33 @@
+#include "script_component.hpp"
+/*
+ * Author: Seb
+ * Renames a group to a given string (groupID), whilst checking that it is not an invalid name.
+ *
+ * Arguments:
+ * 0: The group to be renamed <GROUP>
+ * 1: The new name of the group <STRING>
+ *
+ * Return Value:
+ * Whether the group was succesfully renamed <BOOL>
+ *
+ * Example:
+ * [group player, "leet squad"] call ace_interaction_fnc_renameGroup
+ *
+ * Public: No
+ */
+
+params [
+    ["_group", grpNull, [grpNull]],
+    ["_newName", "", [""]]
+];
+if (_newName isEqualTo (groupID _group)) exitWith {true};
+
+private _nameAlreadyTaken = (toLower _newName) in (allGroups apply {toLower (groupID _x)});
```

Comment:
`allGroups findIf {_newName == groupID _x} == -1`
? I think that'd be faster, but speed doesn't matter much here.

Reply:
I might as well do that as I need to do some more changes; I realised that I need to filter by group side as well as different sides can have the same groupID.